### PR TITLE
update: Thanos retention rules

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Comment by harshini-rangaswamy** [comment]
Thanos is on hold indefinitely.

File: docs/products/metrics/concepts/retention-rules.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  for tracking long-term trends. It is less detailed but ideal for strategic insights
+  and broad analyses.
+
+
+## Downsampling
+
+Downsampling in Aiven for Metrics restructures data series to lower resolution without
+losing accuracy. This approach improves the efficiency of queries spanning large
+time frames. Aiven for Metrics systematically uses this process to aggregate data
+into broader intervals (5 minutes and 1 hour). This method makes query operations
```
Comment (axeoman) [inline docs/products/metrics/concepts/retention-rules.md:36]: not sure if "broader intervals" is the right term. I would mention "resolution" somehow instead

**Review by axeoman — COMMENTED** [review]
Overall looks good: a couple of notes

File: docs/products/metrics/concepts/retention-rules.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/metrics/concepts/retention-rules.md]: Fixed.

File: docs/products/metrics/concepts/retention-rules.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Available configuration options:
+
+- **Raw data**: Retains your metrics data at the highest level of detail,
+  which is crucial for comprehensive analysis where each data point matters.
+- **5-minute downsampled data**: This approach aggregates metrics into 5-minute
+  intervals. It balances detail and storage efficiency, facilitating straightforward
+  analysis over extended periods without significantly affecting data fidelity.
+- **1-hour downsampled data**: This view groups metrics into 1-hour intervals, suitable
+  for tracking long-term trends. It is less detailed but ideal for strategic insights
+  and broad analyses.
```
[the remark this answers is not in the document]
Comment (harshini-rangaswamy) [reply docs/products/metrics/concepts/retention-rules.md]: Update the section to include `compactor.retention.days`

**Pull request closed without merging**